packages/adapter-node/index.js

```diff
@@ -18,6 +18,7 @@ export default function ({
 		host: host_env = 'HOST',
 		port: port_env = 'PORT',
 		origin: origin_env = 'ORIGIN',
+		xffDepth: xff_depth_env = 'XFF_DEPTH',
```

Comment:
No objections to this because I know it's consistent with the others, but it really seems unnecessary

Reply:
i guess the question becomes 'why is it necessary for the others but not this?' and if it _isn't_ necessary for the others, then why do we need to make the env vars configurable at all?